Documentation/prereqs.md

```diff
@@ -0,0 +1,51 @@
+.NET Core native prerequisites
+==============================
+
+This document outlines the dependencies needed to run .NET Core tools and applications. 
+
+## Windows dependencies
+On Windows, the only dependency is the VC++ Redistributable. Depending on the version of Windows you are running on, the versions are changing.
```

Comment:
CRT Dependency: 

On Pre-Win10 OS, the only dependency is uCRT Windows update that can be installed from https://www.microsoft.com/en-us/download/details.aspx?id=48234. You can point folks to https://blogs.msdn.microsoft.com/vcblog/2015/03/03/introducing-the-universal-crt/ for details. 

On Win10+, this is available out of the box.

OS Dependency:

You should call out that for curated OS SKUs (e.g. the ones that are not full server or client - like Nano), they need to have Reverse Forwarders in their OS image.